Makefile

```diff
@@ -278,10 +278,9 @@ SKIP_CREATE_MGMT_CLUSTER ?= false
 
 # Install tools
 
-GINKGO_VER := v2.13.2
 GINKGO_BIN := ginkgo
 GINKGO := $(LOCALBIN)/$(GINKGO_BIN)
-GINKGO_PKG := github.com/onsi/ginkgo/v2/ginkgo
+GINKGO_PKG := $(shell sed '/github.com\/onsi\/ginkgo\/v2 /!d; s/ /@/' go.mod)
```

Comment:
It does assume there is only one instance of `github.com/onsi/ginkgo/v2 ` in go.mod, which should be the case.